src/components/edit/StyledTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+    styled,
+    Theme,
+    Tooltip,
+    tooltipClasses,
+    TooltipProps,
+} from "@mui/material";
+import { StandardCSSProperties } from "@mui/system";
+import React from "react";
+
+export type TooltipStylesInput = (theme: Theme) => StandardCSSProperties;
+
+// crazy nonsense: https://mui.com/material-ui/react-tooltip/#customization
+// tl;dr, tooltip needs to be styled in a very particular way that overrides its own classname
+// and also targets its child elements. it's very easy to miss so making it a utility
+export const makeStyledTooltipMenu = (styleInput: TooltipStylesInput) => {
```

Comment:
tooltips are styled crazily in MUI5 - this is trying to hide that from the rest of the code